src/components/App.js

```diff
@@ -188,12 +186,8 @@ function App({state, UI}) {
                                             }>
                                             <Graph
                                               width={500}
-                                              diffs={getJourneyStopDiffs(
-                                                selectedJourneyData.departures
-                                              )}
-                                              speedAverages={getJourneyAverageSpeeds(
-                                                selectedJourneyData.events
-                                              )}
```

Comment:
Moved these to the Graph component as I don't think the App components needs to know how to calculate diffs and speeds. It is also a performance improvement since I can use useMemo to prevent them from being recalculated on every render.